src/core/dbcsr_lib.F

```diff
@@ -39,25 +64,99 @@ MODULE dbcsr_lib
    CHARACTER(len=*), PARAMETER, PRIVATE :: moduleN = 'dbcsr_lib'
 
    PUBLIC :: dbcsr_init_lib, dbcsr_finalize_lib, dbcsr_clear_mempools
+   PUBLIC :: dbcsr_init_timer,dbcsr_kill_timer, dbcsr_print_timer
```

Comment:
kill? I would propose remove instead. BTW, why these functions are public? They should be private...

Reply:
Ok, I will make them private. Kill sounds better. But remove maybe is not very correct, maybe `dbcsr_finalize_timer`

Reply:
Oh yes, finalize is better